proto/protovalidate/buf/validate/validate.proto

```diff
@@ -209,6 +201,8 @@ message FieldConstraints {
 // Specifies how FieldConstraints.ignore behaves. See the documentation for
 // FieldConstraints.required for definitions of "populated" and "nullable".
 enum Ignore {
+  // buf:lint:ignore ENUM_NO_ALLOW_ALIAS
```

Comment:
Can you do this in the `buf.yaml`?

Reply:
I don't want to disable it for the whole file, just this one enum. Similar discussion as nolint comments vs .golangci.yaml.